lib/chef/resource/windows_pagefile.rb

```diff
@@ -44,11 +44,21 @@ class WindowsPagefile < Chef::Resource
       end
       ```
 
+      **Switch to system managed pagefiles**:
+
+      ```ruby
+      windows_pagefile 'Change the pagefile to System Managed' do
+        path 'E:\pagefile.sys'
+        system_managed true
```

Comment:
The attribute name is `automatic_managed` for this - I like `system_managed` better, but it should be the same in either case...

Reply:
Truth but there are ~3 states for a pagefile : 1) Auto managed, the OS handles pagefiles for all drives. 2) System Managed where the OS handles pagefiles on a drive by drive basis that you specify. 3) You Managed where you explicitly set the pagefile and sizes for all the drives you care about. 4) A combo of you-managed and system managed.